modules/draco/src/index.ts

```diff
@@ -25,8 +25,10 @@ export const DracoWriterWorker = {
   name: 'Draco compressed geometry writer',
   module: 'draco',
   version: VERSION,
+  worker: true,
   options: {
-    draco: {}
+    draco: {},
+    source: null
```

Comment:
I see source here, and workerSource in other places?

Reply:
It is correspond to the property `source` in `worker-utils`. `workerSource` is  used in `tile-converter` to preliminary load worker sources used by the tile converter.